spec/01-unit/010-router_spec.lua

```diff
@@ -95,14 +95,14 @@ describe("Router", function()
 
     it("[uri]", function()
       -- uri
-      local match_t = router.select("GET", "/my-api")
+      local match_t = router.select("GET", "/my-api", "domain.org")
```

Comment:
dully noted - my intent is to replace those with `example.org` in a later contribution, as many tests (outside of the scope of this PR) already use `domain.com` :( (I put them there in the first place 😅)